test/validate.spec.js

```diff
@@ -316,6 +316,13 @@ describe('screener-runner/src/validate', function() {
             expect(err.message).to.equal('child "states" fails because ["states" at position 0 fails because [child "shotsIndex" fails because ["shotsIndex" must be larger than or equal to 0]]]');
           });
       });
+
+      it('should error when url is not prefixed by http', function() {
```

Comment:
@xwa130saucer can you also add a test that verifies that both http and https are valid and go through?